### PR TITLE
Add default, hover, disabled, and size states to storybook

### PR DESCRIPTION
**Pull request by aneyzberg**
This updates the button states for Storybook showing hover, disabled, default, small button size, and large button size.

### CODE REVIEW

**Comment by aneyzberg** [comment]
@camwest I think you need to run `npm run compile` in your root directory, before launching storybook, to recompile the react package.

On a side now, it might be worth adding a development watcher that will run that stuff with a single command so that we do not have to do so manually. I can look into that at some point this week, if it seems like it's worth doing.

File: packages/react/src/2016-12-01/button.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ class Button extends React.Component {
   }
 
   registerListeners(el) {
+    if (el === null) { return; }
```
Comment (camwest) [inline packages/react/src/2016-12-01/button.js]: Under what circumstances is el null?

File: packages/storybook/stories/index.js [hunk 2]
```diff
@@ -14,44 +14,49 @@ See the License for the specific language governing permissions and
...
     };
 
     const sources = [
       {
         label: 'React',
         source: `
-          <OrionButton
-            backgroundColor={"${props.backgroundColor}"}
+          <Button
+            background={"${props.background}"}
```
Comment (camwest) [inline packages/storybook/stories/index.js]: Don't use {} around static props. 

```
background="${props.background}"
```

Works just fine

File: packages/storybook/stories/index.js [hunk 3]
```diff
@@ -14,44 +14,49 @@ See the License for the specific language governing permissions and
...
             color={"${props.color}"}
             onClick={action('clicked')}
           >
             {"${buttonText}"}
-          </OrionButton>
+          </Button>
         `,
       }, {
         label: 'Web components',
```
Comment (camwest) [inline packages/storybook/stories/index.js]: I don't think we should document the Web Component variant since it doesn't actually work yet.

**Comment by nfiniteset** [comment]
@aneyzberg Mind removing storybook/components/button.js as a part of this PR?

**Comment by aneyzberg** [comment]
@nfiniteset done.

**Comment by camwest** [comment]
@aneyzberg thanks for the submission! One last note - don't squash between submissions. We'll squash and merge before putting the branch into master and it messes up code review history.

**Pull request merged**